src/Frontend/test/specs/licensing/trial-license-notifications.spec.ts

```diff
@@ -31,6 +34,8 @@ describe("FEATURE: Trial license notifications", () => {
 
         //This has to use waitFor because of shared state between test runs. See issue documented issue and proposed solution https://github.com/Particular/ServicePulse/issues/1905
         await waitFor(() => expect(screen.queryByRole("status", { name: /trial license bar information/i })).toBeNull());
+
+        await flushPromises();
```

Comment:
I wonder if this can be moved to:
https://github.com/Particular/ServicePulse/blob/add-dashboard-acceptance-tests/src/Frontend/test/drivers/vitest/driver.ts#L49

Reply:
I'm not sure we would have to test.

Reply:
I think it could be moved, but I don't think I want to make that change in this PR.